shared/assay_rnaSeq_metadata_template.json

```diff
@@ -0,0 +1,58 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id":"sysbio.metadataTemplates-assay.rnaSeq",
+    "description": "SysBio rnaSeq metadata template schema",
+    "properties": {
+        "specimenID": {
+            "$ref": "sage.annotations-experimentalData.specimenID"
+        },
+        "libraryID": {
+            "$ref": "sage.annotations-ngs.libraryID"
+        },
+        "assay": {
+          "$ref": "sage.annotations-experimentalData.assay"
+        },
+        "platform": {
+            "$ref": "sage.annotations-experimentalData.platform"
+        },
+        "RIN": {
```

Comment:
What is the difference between "RIN" and "organRIN"?

Reply:
This is the definition I see for organRIN: "A general quality metric of the organ or tissue, not necessarily a direct RIN measure of the sample used in data generation."

This is the definition I see for RIN: "RNA integrity number"

I think one is specific to the exact sample being used, while the other is a general measure for the whole organ.